src/Microsoft.Identity.Client/Platforms/iOS/TokenCacheAccessor.cs

```diff
@@ -165,6 +165,11 @@ private SecStatusCode SetValueForKey(string key, string value, string service)
 
             var result = SecKeyChain.Add(CreateRecord(key, value, service));
 
+            if (result != SecStatusCode.Success)
+            {
+                _requestContext.Logger.Warning("Failed to add cache record to iOS Keychain: SecStatusCode." + result.ToString());
```

Comment:
you might want to reformat the log message ...`SecStatusCode: " + result.ToString());` to have a space between the end of the sentence and the result.

Reply:
That was my first thought but I changed it to the current format because I want the error message to come out as "SecStatusCode.Param". But now that I think about it "SecStatusCode: Param". is fine